SU2_CFD/src/solver_direct_mean_inc.cpp

```diff
@@ -6238,6 +6295,21 @@ CIncNSSolver::CIncNSSolver(CGeometry *geometry, CConfig *config, unsigned short
     node[iPoint]->SetLaminarViscosity(Viscosity_Inf);
   }
 
+  /*--- Initialize the BGS residuals in FSI problems. ---*/
+  if (fsi){
+    Residual_BGS      = new su2double[nVar];         for (iVar = 0; iVar < nVar; iVar++) Residual_RMS[iVar]  = 0.0;
```

Comment:
Please double check that these extra arrays are deleted in the constructors.

Reply:
In CSolver::~CSolver(void), lines 114-116:

  if (Residual_BGS != NULL) delete [] Residual_BGS;
  if (Residual_Max_BGS != NULL) delete [] Residual_Max_BGS;
  if (Point_Max_BGS != NULL) delete [] Point_Max_BGS;